packages/realm/src/PropertyMap.ts

```diff
@@ -31,7 +31,10 @@ export class PropertyMap {
   private objectSchemaName: string | null = null;
   private initialized = false;
   private mapping: Record<string, PropertyHelpers | undefined> = {};
-  private nameByColumnKey: Map<binding.ColKey, string> = new Map();
+  /**
+   * Note: Cannot key by the binding.ColKey directly, as this is `Long` on JSC (which does not compare equal)
```

Comment:
Also, could you clarify `which does not compare equal`?
```suggestion
   * Note: Cannot get key by the binding.ColKey directly, as this is `Long` on JSC (which does not compare equal)
```

Reply:
I actually think "Cannot key by" is correct. It's talking about key'ing a record / object by some sort of value. At least that's an expression I've seen before 🙂
```suggestion
   * Note: Cannot key by the binding.ColKey directly, as this is `Long` on JSC (which does not pass equality checks like `bigint` does)
```

Reply:
> which does not compare equal

What I meant is that two instances of `Long` representing the same logical value are not `===` equal.

Reply:
Thanks 👍